app/controllers/bets_controller.rb

```diff
@@ -1,9 +1,13 @@
 class BetsController < ApplicationController
+  include Utilities::Paginator
+  helper_method :paginator
+
   def index
     @items_per_page = 25
     @current_page = params[:page] || '1'
     @bets = Bet.offset(@items_per_page * (@current_page.to_i - 1))
                .limit(@items_per_page)
+    @number_of_all_bets = Bet.all.count
```

Comment:
I think I would like to see some of the logic from lines 8 - 10 moved into the paginator somehow.

In general we try and avoid computation in controllers. They are really only for loading resources and rending things